src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -180,16 +186,42 @@ def repro(self, repro_timeout, repro_args):
     """
     logs.log('Running Syzkaller testcase.')
     additional_args = copy.copy(repro_args)
-    result = self.run_and_wait(additional_args, timeout=repro_timeout)
-    result.return_code = self._crash_was_reproducible(result.output)
-
-    if result.return_code:
-      logs.log('Successfully reproduced crash.')
-    else:
-      logs.log('Failed to reproduce crash.')
-    logs.log('Syzkaller repro testcase stopped.')
-    return engine.ReproduceResult(result.command, result.return_code,
-                                  result.time_executed, result.output)
+
+    for retry_count in range(REPRO_RETRY_MAX):
+      result = self.run_and_wait(additional_args, timeout=repro_timeout)
+      log_location = re.search(REPRODUCE_LOG_LOCATION_PATTERN, result.output)
+
+      # syz-crush stopped before capturing crash output
+      if not log_location:
+        continue
+
+      # syz-crush did not reproduce any crash
+      if not self._crash_was_reproducible(result.output):
+        continue
+
+      _type, log_index, log_dir = log_location.groups()  # pylint: disable=invalid-name
+
+      try:
+        with open(f'{log_dir}/reproducer{log_index}', 'r') as f:
```

Comment:
nit: use `os.path.join` here instead of hardcoding '/'.

Reply:
👍